src/collapse/perf.rs

```diff
@@ -245,7 +262,20 @@ impl PerfState {
                 }
             }
 
-            // TODO: show_inline
+            // --inline flag
+            if self.opt.show_inline && can_un_inline(module) {
+                match self.un_inline(pc, module) {
```

Comment:
How about:
```rust
if let Err(e) = self.un_inline(pc, module) {
    // ...
} else {
    return;
}
```

Reply:
Actually, on second thought, I wonder if we should just log directly from inside `self.un_inline`, since it can probably provide more specific error messages. It also saves us from having to do an additional layer of error handling here.